featuretools/tests/entityset_tests/test_es.py

```diff
@@ -961,3 +961,31 @@ def _slice_for(es, filter_eid, time_last=None):
                                     index_eid='customers',
                                     instances=[0],
                                     time_last=time_last)
+
+
+def test_same_index_values():
+    transactions_df = pd.DataFrame({"id": [1, 2, 3, 4, 5, 6],
+                                    "session_id": [1, 2, 1, 3, 4, 5],
```

Comment:
So we don't need `session_id`, `amount`, and `fraud` in the dataframe. Since these columns are used anywhere, we should take them out. We should just have a dataframe with the columns we use for testing.